src/zypper/services/rhsm

```diff
@@ -72,16 +75,42 @@ class RepoLocker(Locker):
 
 
 class ZypperService(object):
+
+    ZYPP_RHSM_PLUGIN_CONFIG_FILE = '/etc/rhsm/zypper.conf'
+
     def __init__(self):
         logutil.init_logger()
         init_dep_injection()
         self.is_root = os.getuid() == 0
-        # Read subscription-manager config
+        self.full_refresh_on_yum = True
+        self.ca_cert_dir = '/etc/rhsm/ca/'
```

Comment:
I see that these two attributes (`full_refresh_on_yum` and `ca_cert_dir`) are overridden later. I assume you were just making sure these properties are populated in the `__init__` method? I would rather see them set to `None` in `__init__` (because this prevents duplication of default value in the code, and prevents confusion).

Reply:
Good point.